examples/va/hci/values.yaml

```diff
@@ -0,0 +1,21 @@
+# local-config: referenced, but not emitted by kustomize
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: edpm-values-post-ceph
+  annotations:
+    config.kubernetes.io/local-config: "true"
+data:
+  nodeset:
+    services:
+    - ceph-client
+    - ovn
+    - neutron-metadata
+    - libvirt
+    - nova-custom-ceph
+  ceph:
+    conf: CHANGEME_CEPH_CONF
```

Comment:
I think it's fine to keep this `hci/values.yaml` as it, but I wanted to point out that if the ci-framework wants to generate a populated version of this file that the ceph values are known it from here:

https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/cifmw_ceph_client/templates/k8s_ceph_secret.yml.j2